### DISCUSSION TITLE
SQL plugin retrieve data

### DISCUSSION THREAD

**Discussion by noe-sautel** [jgraph/drawio#2806] [Q&A] [answered]
We can use the SQL plugin to create a table for an entity relation model. It is very useful if we have columns name into a text document because we can simply copy paste it and add param for each column.

However, I would like to know how can I see the original request of my table ? Imagine I just lost my text document where my 100+ columns were stored and I want to update a large part of them on draw.

**Comment by noe-sautel** [comment, marked as answer]
Few minutes later, I realized I can use the text plugin to retrieve text from object. By selecting columns then use the plugin, I can get the columns in text format that I wanted

**Reply by lastlink** [reply]
@noestl  I've added an exportsql plugin that will export the create table sql statments. #3092

**Reply by noe-sautel** [reply]
Thank you for taking the time @lastlink to improve it. I will check again on draw when PR will be approved

**Reply by lastlink** [reply]
Updated pr #4122
